### PR TITLE
Feat/81 comment types

### PR DESCRIPTION
**Pull request by jshor**
adds different styling to the server-side comments/status changes

### CODE REVIEW

File: client/static/main.css [hunk 1]
```diff
@@ -177,6 +177,18 @@ Comments Page
   padding: 0px 16px;
   width: 80%;
 }
+.status-change {
+  border-left: 2px solid #808080;
+}
+.status-change, .comment{
+  justify-content: flex-start !important;
```
Comment (jackofseattle) [inline client/static/main.css]: Why does this need to be important?

File: client/vivace/comments/controller.js [hunk 2]
```diff
@@ -34,6 +34,12 @@ module.exports = [
 
       .then(/* istanbul ignore next */(data) => {
         this.comments = data;
+        this.comments.push({
```
Comment (jackofseattle) [inline client/vivace/comments/controller.js]: Looks like you left in some test code.

File: client/static/main.css (hunk 1 above)
Comment (jshor) [inline client/static/main.css]: @jackofseattle this class is supposed to be used with `md-whiteframe` which has `justify-content: center` set - the `!important` flag is necessary to override it

**Comment by jackofseattle** [comment]
LGTM

**Pull request merged**